<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MYIMGTOOLClass</class>
 <widget class="QMainWindow" name="MYIMGTOOLClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MYIMGTOOL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-background-color: rgb(0, 170, 255);</string>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>搜索关键字</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>新建档案</string>
     </property>
     <addaction name="action_new_ver1"/>
     <addaction name="action_new_ver2"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="action_open"/>
    <addaction name="action_close"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_operations">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="action_import"/>
    <addaction name="action_import_folder"/>
    <addaction name="action_export"/>
    <addaction name="action_rebuild"/>
   </widget>
   <widget class="QMenu" name="menu_misc">
    <property name="title">
     <string>杂项</string>
    </property>
    <addaction name="action_showinfo"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_operations"/>
   <addaction name="menu_misc"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="statusTip">
    <string>打开一个IMG文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_close">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
   <property name="statusTip">
    <string>关闭当前打开的文件</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_import">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>导入文件...</string>
   </property>
   <property name="toolTip">
    <string>导入文件</string>
   </property>
   <property name="statusTip">
    <string>导入一个或多个文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_rebuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>重建IMG</string>
   </property>
   <property name="toolTip">
    <string>重建IMG</string>
   </property>
   <property name="statusTip">
    <string>重建IMG以去除无用空间(需要一段时间)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_import_folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>导入文件夹...</string>
   </property>
   <property name="toolTip">
    <string>导入文件夹</string>
   </property>
   <property name="statusTip">
    <string>导入文件夹内的所有文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_showinfo">
   <property name="text">
    <string>关于...</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
   <property name="statusTip">
    <string>显示本工具的信息</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>导出所选文件...</string>
   </property>
   <property name="toolTip">
    <string>导出所选文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_new_ver1">
   <property name="text">
    <string>版本1...</string>
   </property>
   <property name="toolTip">
    <string>新建版本1档案</string>
   </property>
  </action>
  <action name="action_new_ver2">
   <property name="text">
    <string>版本2...</string>
   </property>
   <property name="toolTip">
    <string>新建版本2档案</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
